docs/content/posts/2023-11-13-release-1.17.2.md

```diff
@@ -0,0 +1,198 @@
+---
+title:  "Apache Flink 1.17.2 Release Announcement"
+date: "2023-11-13T00:00:00.000Z"
```

Comment:
Am I right that date should be change before the announcement?

Reply:
Yes, it will change before the announcement.